App.js

```diff
@@ -52,44 +55,57 @@ const HomeScreen = ({ navigation }) => {
+                        }}
+                    />
+                </StyledLogo>
+                <StyledTagline style={{ paddingTop: 10, paddingBottom: 20 }}>The game that gets families talking!</StyledTagline>
 
-                    <StyledHomeButtonView>
-                        <HomeButton
-                            text="PLAY GAME"
-                            onPress={() => navigation.navigate('Play')}
-                            source={require('./assets/genconnect_ombre_allaboutme.png')}
-                        />
-                        <HomeButton
-                            text="HOW TO PLAY"
-                            onPress={() => navigation.navigate('How To Play')}
-                            source={require('./assets/genconnect_ombre_brightfuture.png')}
-                        />
-                        <HomeButton
-                            text="PARENT GUIDE"
-                            onPress={() => navigation.navigate('Parent Guide')}
-                            source={require('./assets/genconnect_ombre_whatwouldyoudo.png')}
-                        />
-                        <HomeButton
-                            text="PARENT TIPS"
-                            onPress={() => navigation.navigate('Parent Tips')}
-                            source={require('./assets/genconnect_ombre_dancechallenge.png')}
-                        />
-                    </StyledHomeButtonView>
-                    <StatusBar style="auto" />
-                </ScrollView>
+                <StyledHomeButtonView>
+                    <HomeButton
+                        text="PLAY GAME"
+                        onPress={() => navigation.navigate('Play')}
+                        source={require('./assets/genconnect_ombre_allaboutme.png')}
+                    />
+                    <HomeButton
+                        text="HOW TO PLAY"
+                        onPress={() => navigation.navigate('How To Play')}
+                        source={require('./assets/genconnect_ombre_brightfuture.png')}
+                    />
+                    <HomeButton
+                        text="PARENT GUIDE"
+                        onPress={() => navigation.navigate('Parent Guide')}
+                        source={require('./assets/genconnect_ombre_whatwouldyoudo.png')}
+                    />
+                    <HomeButton
+                        text="PARENT TIPS"
+                        onPress={() => navigation.navigate('Parent Tips')}
+                        source={require('./assets/genconnect_ombre_dancechallenge.png')}
+                    />
+                </StyledHomeButtonView>
+                <StatusBar style="auto" />
+                <TouchableOpacity
+                    onPress={() => navigation.navigate('Information')}
+                    style={{
+                        position: 'absolute',
+                        right: 20,
+                        top: 40,
+                    }} >
```

Comment:
same here

Reply:
i don't think i did this one, what's the best way to resolve this @sukritsangvong